src/components/AccessibilityDocumentsList/index.tsx

```diff
@@ -23,24 +37,69 @@ type Document = {
 type DocumentsListProps = {
   documents: Document[];
   requestName: string;
+  requestId: string;
 };
 
 const AccessibilityDocumentsList = ({
   documents,
-  requestName
+  requestName,
+  requestId
 }: DocumentsListProps) => {
   const { t } = useTranslation('accessibility');
+  const [isModalOpen, setModalOpen] = useState(false);
 
   const getDocType = (documentType: {
     commonType: AccessibilityRequestDocumentCommonType;
-    otherTypeDescription: string;
+    otherTypeDescription: string | null;
   }) => {
     if (documentType.commonType !== 'OTHER') {
       return translateDocumentType(
         documentType.commonType as AccessibilityRequestDocumentCommonType
       );
     }
-    return documentType.otherTypeDescription;
+    return documentType.otherTypeDescription || '';
+  };
+
+  const [mutate] = useMutation<
+    DeleteAccessibilityRequestDocument,
+    DeleteAccessibilityRequestDocumentVariables
+  >(DeleteAccessibilityRequestDocumentQuery);
+
+  const submitDelete = (id: string) => {
+    mutate({
+      variables: {
+        input: {
+          id
+        }
+      },
+      update(cache) {
```

Comment:
This is an attempt to update the cache after the mutation instead of just refetching the data from the server. Is this worth it?